common/data/details.js

```diff
@@ -22152,6 +22152,33 @@ window.details = {
             },
         ],
         potentialNotes: "#{enrage}",
+        festAbility: [
```

Comment:
This file isn't used anymore for PvP data. It will be removed soon. Instead, enter PVP data in rumble.json

Reply:
Got it. Should have noticed. I will double check with the units that have already the data on the JSON file and add the ones that dont have it in the coming days.